app-dicts/dictd-devils/Manifest

```diff
@@ -1 +1 @@
-DIST devils-dict-pre.tar.gz 175823 BLAKE2B caf11573aa4a8e1774a15a54eab7ed0a9069588498d51d382e030da7dccca2fb7e03170506becfcdb8246f8802655bca22abe568d839cee608e5310951328d93 SHA512 b085f0b8e8ead1070f31fdcfff7489b09550a92c50fb5e4018f2a953b456871900eee65e91f916e4be155ded20cb906d683bdc5fb5b5370201ce664d8fd8c40d
```

Comment:
Sure, in theory.  That removal was mostly due to the older versions being masked and (then) approaching removal due to that anyway; I figured this would be serving dual purpose of both introducing new and fulfilling that warning.  If it's better to extend their life for the transition, then I have no problem with that.

However, those versions have already been removed upstream -- and I'm rebasing the PR as part of fixing everything -- so by *not* removing them here, I'd actually be adding old EAPI 0 ebuilds back into the tree.  I'd rather not do that.